projects/addon-editor/components/color-selector/color-selector.component.ts

```diff
@@ -188,7 +188,7 @@ export class TuiColorSelectorComponent {
 
     private getGradient(direction: TuiGradientDirection): string {
         return `linear-gradient(${direction}, ${[...this.stopsKeys]
-            .sort()
+            .sort((a, b) => a - b)
```

Comment:
Maybe `tuiDefaultSort`? Didn't look too deep into it :)

Reply:
fixed